test/integration/test_build.py

```diff
@@ -98,9 +98,11 @@ def test_user_python_requirement(cli, runtime, ee_tag, tmp_path, data_dir):
 
 @pytest.mark.test_all_runtimes
 def test_python_git_requirement(cli, runtime, ee_tag, tmp_path, data_dir):
+    # FIXME: is this test even necessary, since we don't (currently) special-case git in builder?
+
```

Comment:
I think you're right and we don't need this test anymore. The older releases have `git` in the `bindep.txt` file for the builder repo (and we still do, actually). Since we aren't building and publishing images anymore, this is a pointless test. Let's remove it (and the test data).